website/content/docs/agent/index.mdx

```diff
@@ -366,6 +366,63 @@ service {
 </Tab>
 </Tabs>
 
+## Client Node with Multiple Interfaces or IP addresses
+
+The following example shows how to configure Consul to listen on multiple interfaces or IP addresses using a [go-sockaddr template].
+
+The `bind_addr` is used for internal RPC and Serf communication (see the Agent Configuration for [bind_addr](/docs/agent/options#bind_addr) for more information).
+
+The `client_addr` configuration specifies IP addresses used for HTTP, HTTPS, DNS and GRPC servers. (see the Agent Configuration for [client_addr](/docs/agent/options#client_addr) for more information).
+
+<Tabs>
+<Tab heading="HCL">
```

Comment:
@clly @webdog  use `<CodeTabs>` instead of `<Tabs>`. Watch this [video](https://hashicorp.atlassian.net/wiki/spaces/NET/pages/2259910751/How+to+contribute+to+Consul+documentation#Web-Components) to see how to use them.

Reply:
I took the liberty of updating the rest of the page to use CodeTabs so they're all consistent